validator/sawtooth_validator/journal/publisher.py

```diff
@@ -763,10 +763,12 @@ def on_check_publish_block(self, force=False):
                         and self._pending_batches):
                     self._build_candidate_block(self._chain_head)
 
-                if self._candidate_block and (
-                    force or
-                    self._candidate_block.has_pending_batches()) and \
-                        self._candidate_block.check_publish_block():
+                # there is no good way to format this
```

Comment:
While this may be true, I don't think we need to call it out with a comment.

Reply:
Do you think there should be no comment at all? My thought was that the thing looks bizarre and warranted a comment pointing out that it isn't a mistake.

Reply:
Either no comment or a comment explaining why it has to look like that.

Reply:
Okay, I took it out.